src/components/RedoIcon.tsx

```diff
@@ -2,21 +2,44 @@ import { css, cx } from '../../styled-system/css'
 import { icon } from '../../styled-system/recipes'
 import { token } from '../../styled-system/tokens'
 import Icon from '../@types/Icon'
+import { ICON_SCALING_FACTOR } from '../constants'
 
 /** A redo icon. */
-const RedoIcon = ({ fill, size = 18, style, cssRaw }: Icon) => {
+const RedoIcon = ({ fill, size = 18, style = {}, cssRaw }: Icon) => {
+  const newSize = size * ICON_SCALING_FACTOR
+  const strokeColor = style.fill || fill || token('colors.fg')
+
   return (
     <svg
-      version='1.1'
       className={cx(icon(), css(cssRaw))}
       xmlns='http://www.w3.org/2000/svg'
-      width={size}
-      height={size}
+      width={newSize}
+      height={newSize}
+      viewBox='0 0 24 24'
       fill={fill || token('colors.fg')}
-      style={style}
-      viewBox='0 -50 600 600'
+      style={{ ...style, width: `${newSize}px`, height: `${newSize}px` }}
```

Comment:
Why set `width` and `height` in the `style` attribute when they are already set in the width and height attributes? It seems like it should be either one or the other.